apps/SageAccountWeb/src/components/SignUpdatedTermsOfUsePage.tsx

```diff
@@ -50,7 +53,11 @@ export const SignUpdatedTermsOfUsePage = (
           },
           {
             onSuccess: () => {
-              restoreLastPlace()
+              // take user back to page they came from in the source app, if stored in a cookie
+              const isProcessed = processRedirectURLInOneSage()
+              if (!isProcessed) {
+                restoreLastPlace(history)
+              }
```

Comment:
Do we set the cookie when we send the user to their account settings page in SWC (for example)?  If so, they will be sent back to SWC after signing instead of to the account settings (which is not ideal)

Reply:
Perhaps not. The redirect issue may only affect localhost, which is annoying but maybe not that bad. I will remove the cookie check for now.